pkg/kp/rollstore/store.go

```diff
@@ -176,3 +190,9 @@ func RollLockPath(rcId rcf.ID) (string, error) {
 
 	return path.Join(kp.LOCK_TREE, subRollPath), nil
 }
+
+func kvpToRU(kvp *api.KVPair) (rollf.Update, error) {
+	ru := rollf.Update{}
+	err := json.Unmarshal(kvp.Value, &ru)
+	return ru, util.Errorf("Unable to unmarshal value as rolling update: %s", err)
```

Comment:
Hmm. So the `error` returned by `kvpToRU` will always be non-nil, even if that returned by `json.Umharshal` is non-nil. I'm unsure of this, especially since this will make  at least `List()` always fail